packages/theme/src/sidebar/sidebar.css.ts

```diff
@@ -25,8 +26,9 @@ export const sidebar = recipe({
       display: 'flex',
       overflowY: 'scroll',
       position: 'relative',
-      top: `${config.appHeader.height}px`,
-      height: 'auto'
+      top: '-20px',
+      // top: `${config.appHeader.height}px`,
+      height: '100%'
```

Comment:
-20px is a magic number, would look at what we can use from the theme.

As before, not covering the app header, creates new issues, as the close button appears twice. Would ask design folks to help us work out the best solution.